locale/fi/typography.json

```diff
@@ -4,7 +4,7 @@
   "note.title": "Saavutettavuus ja käytettävyys",
```

Comment:
The intro seems to be missing. At least there's one in the design.

Reply:
I have now committed the change of missing intro translation for Finnish.